app/models/jurisdiction.rb

```diff
@@ -286,6 +287,10 @@ def template_version_customization(template_version, sandbox = nil)
 
   private
 
+  def inbox_enabled_changed?
```

Comment:
is there a good reason to have a non-standard name for this boolean in addition to the standard one, thus making the codebase one step more crufted up?